rosdep/base.yaml

```diff
@@ -2389,6 +2389,11 @@ libpqxx:
     trusty: [libpqxx-4.0]
     utopic: [libpqxx-4.0]
     vivid: [libpqxx-4.0]
+    wily: [libpqxx-4.0]
+    xenial: [libpqxx-4.0]
+    yakkety: [libpqxx-4.0v5]
+    zesty: [libpqxx-4.0v5]
+    artful: [libpqxx-4.0v5]
```

Comment:
I don't think this makes sense. 17.10 comes after 17.04; it's reasonable for the versions to be in order of release, not in order of name. Otherwise versions will get weirdly interleaved as we move forward, and instead of appending new versions to the end, they will be inserted awkwardly in the middle. For example, we would have Warty in between Vivid and Wily and Hoary in between Hardy and Intrepid, so the version list would do this:

* 15.04
* 4.10
* 15.10
* 16.04
* ...